src/Tizen.Applications.Common/Interop/Interop.RPCPort.cs

```diff
@@ -42,6 +42,26 @@ internal enum PortType
 
         internal static partial class Parcel
         {
+            /// <summary>
+            /// This structure represents the struct timespec.
+            /// </summary>
+            /// <since_tizen> 9 </since_tizen>
+            [StructLayout(LayoutKind.Sequential)]
+            internal struct TimeStamp
```

Comment:
저희가 native 구조체로부터 marshing하기 위해 매핑하여 작성한 struct에 대해 빌드시 사이즈가 동일한지 검사를 하고 있습니다. 이를 위해 NativeStruct attribute를 작성해야합니다.
```suggestion
            [NativeStruct("struct timespec", Include = "time.h")]
            [StructLayout(LayoutKind.Sequential)]
            public struct TimeStamp
```